ssh/ssh_config_scs_general

```diff
@@ -0,0 +1,8 @@
+
+Host scs-jumphost
+   Hostname 153.92.93.119
+   Port 41115
+
+Host scs-manager
+   Hostname 188.244.101.28
+
```

Comment:
You can also use the`osism.commons.sshconfig` role for this. Additional SSH client config can be added with the help of the `sshconfig_extra` parameter (https://github.com/osism/ansible-collection-commons/blob/main/roles/sshconfig/tasks/main.yml#L32-L41).

If necessary, we can add an `sshconfig_extra_files` parameter that copies files stored in the configuration repository directly and stores them in `.ssh/config.d`.

Reply:
This is a general ssh configuration, which is used by users on their own workstation/laptop systems.
(see https://github.com/SovereignCloudStack/hardware-landscape/blob/provision_nodes/documentation/System_Usage.md).